PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/UIHelperImpl.java

```diff
@@ -97,6 +101,9 @@
     public static final Key<String> RESOURCE_ID = new Key<>("resourceId");
     public static final Key<String> WEBAPP_ID = new Key<>("webAppId");
     public static final Key<String> APP_ID = new Key<>("appId");
+    public static final Key<AppResourceInner> SPRING_CLOUD_APP = new Key<>("springCloudApp");
+    public static final Key<DeploymentResourceInner> SPRING_CLOUD_DEPLOYMENT = new Key<>("springCloudAppDeployment");
```

Comment:
is `SPRING_CLOUD_DEPLOYMENT` used somewhere?

Reply:
No need for this const